test/tcp/TcpServer.cpp

```diff
@@ -72,16 +72,16 @@ class ServerRequestHandler : public DefaultRequestHandler {
 
   void handleFireAndForgetRequest(Payload request) override {
     LOG(INFO) << "ServerRequestHandler.handleFireAndForgetRequest " << request
-              << "\n";
+              << std::endl;
```

Comment:
we shouldn't be writing new lines into LOG statements, it would make grep filtering impossible.